### PR TITLE
Add CleanupElement like CefSharp.Wpf.

### PR DESCRIPTION
**Pull request by kkwpsv**
Behave like CefSharp.Wpf.

### CODE REVIEW

**Comment by kkwpsv** [comment]
Should I change CleanupElement, when moving this control to another window? CefSharp.Wpf not handle this situation now.

File: CefSharp.Wpf.HwndHost/ChromiumWebBrowser.cs [hunk 1]
```diff
@@ -1333,6 +1348,70 @@ public double ZoomLevelIncrement
...
+        protected virtual void OnCleanupElementChanged(FrameworkElement oldValue, FrameworkElement newValue)
+        {
+            if (oldValue != null)
+            {
+                oldValue.Unloaded -= OnCleanupElementUnloaded;
+            }
+
+            if (newValue != null)
+            {
+                newValue.Unloaded -= OnCleanupElementUnloaded;
```
Comment (amaitland) [inline CefSharp.Wpf.HwndHost/ChromiumWebBrowser.cs:1399]: This was probably a mistake and should be removed.

**Comment by amaitland** [comment]
> Should I change CleanupElement, when moving this control to another window? CefSharp.Wpf not handle this situation now.

Yes, when the presentation source changes if the type is a window.

File: CefSharp.Wpf.HwndHost/ChromiumWebBrowser.cs (hunk 1 above)
Comment (kkwpsv) [reply CefSharp.Wpf.HwndHost/ChromiumWebBrowser.cs]: Same as CefSharp.Wpf. And it's redundant. Removed.

Comment (amaitland) [reply CefSharp.Wpf.HwndHost/ChromiumWebBrowser.cs]: Should be removed there also.

**Comment by amaitland** [comment]
I don't really have time to test this at the moment, so I'll comment the `PresentationSourceChangedHandler` changes out for now, you can manually assign `CleanupElement` in `xaml`. Changes can be uncommented once they have been further tested.

**Pull request merged**